### DISCUSSION TITLE
Fatal compilation on Windows

### DISCUSSION THREAD

**Discussion by LEley22** [nrc-cnrc/EGSnrc#1029] [General]
Hi!

I've just installed EGSnrc for the first time, and I'm trying to compile the cavity 'example' (?), but I am getting a fatal error immediately which states 'not a git repository (or any of the parent directories).git, which prints twice.

I followed the Windows installation instructions which all seemed to go fine, I did not get any error messages during this process at all so I'm not too sure what the issue is! Any advice would be massively appreciated.

Thank you,
Lauryn

**Comment by ftessier** [comment]
Please consider the modifications applied in #972. We will release a 2023a version of EGSnrc soon to address this issue.

**Reply by ftessier** [reply]
Alternatively, you can also run [EGSnrc in a linux virtual machine](https://github.com/nrc-cnrc/EGSnrc/wiki/Install-EGSnrc-on-VirtualBox) for the time being.
